packages/annotations/src/ui-components/comment.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="alpheios-annotations-comment__container" data-annotation-container>
+    <div class="alpheios-annotations__act-panel">
+      <div class="alpheios-annotations__act-ctrls" :data-annotation-form-open="isFormOpen">
+        <div class="alpheios-annotations__act-ctrls-edit" @click.stop="editComment"
+             :data-annotation-selected="isEditing">[*comment]</div>
+        <div class="alpheios-annotations__act-ctrls-remove" @click.stop="removeComment"
+             :data-annotation-selected="isRemoving">[-remove]</div>
+      </div>
+
+      <div class="alpheios-annotations__act-form" v-show="isEditing">
+        <div class="alpheios-annotations__act-form-content">
+          <div class="alpheios-annotations__act-form-headline">Edit the comment:</div>
+          <div class="alpheios-annotations__act-form-text-input-group">
+            <textarea id="lemma-definition" cols="50"
+                      name="lemma-definition" rows="4">A comment about the part of speech of the lexeme</textarea>
+          </div>
+        </div>
+
+        <div class="alpheios-annotations__act-form-ctrls">
+          <div class="alpheios-annotations__act-form-ctrls-save" @click.stop="closeForm">Save</div>
```

Comment:
I know it's just a prototype but I think I would still like for all of the text to be in messages files (Save, Remove, Add, Cancel, etc.)

Reply:
That's a good idea. will add that.